### DISCUSSION TITLE
Caddyfile config

### DISCUSSION THREAD

**Discussion by hvisage** [poweradmin/poweradmin#421] [Show and tell]
For those interested, here is the config file that works for me.
I extracted the poweradmin sources to `/srv/www`
`php-fpm.sock` is a link via the alternatives method of Debian, so make sure that is set for you accordingly.

Also, I only needed port 80, no SSL, as I have another SSL frontend (also Caddy :) ) that does my URL/host routing and reverse proxy

```
:80 {
	log {
		output file /var/log/caddy/caddy.log {
		}
	}

	# Set this path to your site's directory.
	root * /srv/www

	rewrite /update /dynamic_update.php
	rewrite /nic/update /dynamic_update.php

	php_fastcgi * unix//run/php/php-fpm.sock {
	}

	file_server * {
	}
}
```

**Comment by edmondas** [comment]
Thanks for letting me know. I don't use this web server much myself, but I've added your provided example to the documentation ([manual installation section](https://docs.poweradmin.org/installation/manual/#caddy-configuration)). I appreciate your contribution!
